src/getHumanAge.test.js

```diff
@@ -7,4 +7,25 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should convert cat and dog ages correctly', () => {
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
```

Comment:
The test case for converting cat and dog ages of 27 years each to human years is incorrect according to the task description. For 27 cat years, the expected human age should be 3, and for 27 dog years, it should be 2. Update the expected result to `[3, 2]`.